elasticsearch-transport/lib/elasticsearch/transport/transport/base.rb

```diff
@@ -273,6 +275,10 @@ def perform_request(method, path, params={}, body=nil, &block)
           @last_request_at = Time.now
         end
 
+        def deserialize_response(response)
```

Comment:
Is this method part of the public API? If not (which I suspect), I have used the Python-inspired convention of prepending the method name with two underscores (`__deserialize_response`), to better distinguish between them. What do you think about it?